public/app/features/explore/LogsContainer.tsx

```diff
@@ -78,7 +78,6 @@ export class LogsContainer extends PureComponent<LogsContainerProps> {
           data={logsResult}
           dedupedData={dedupedResult}
           exploreId={exploreId}
-          key={logsResult && logsResult.id}
```

Comment:
Removed `id` from `LogsModel` since I don't think the key is needed here - to me should only be needed if you're rendering multiple `<Logs />` components.